File: examples/android-imageview/app/src/main/java/com/cloudinary/android_imageview/MainActivity.java
package com.cloudinary.android_imageview;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.bumptech.glide.Glide;
import com.cloudinary.android.MediaManager;
import com.cloudinary.android_imageview.databinding.ActivityMainBinding;

import java.util.HashMap;
import java.util.Map;

public class MainActivity extends AppCompatActivity  {

    ActivityMainBinding binding;

    ImageView myImageView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        // Get image view from xml.
        myImageView = binding.myImageview;

        initMediaManager();

        String url = MediaManager.get().url().generate("/samples/animals/cat");

        // Use glide to load image from url into imageview
        Glide.with(this).load(url).into(myImageView);
    }

    private void initMediaManager() {
        Map config = new HashMap();
        config.put("cloud_name", "dkxtn2tlm");
        MediaManager.init(this, config);
    }
}

File: examples/kotlin-imageview/app/src/main/java/com/cloudinary/kotlin_imageview/MainActivity.kt
package com.cloudinary.kotlin_imageview

import android.os.Bundle
import android.widget.ImageView
import androidx.activity.ComponentActivity
import androidx.core.view.WindowCompat
import com.bumptech.glide.Glide
import com.cloudinary.Cloudinary
import com.cloudinary.config.CloudinaryConfig
import com.cloudinary.kotlin_imageview.databinding.ActivityMainBinding

class MainActivity: ComponentActivity() {

    private lateinit var binding: ActivityMainBinding
    private lateinit var myImageView: ImageView;

    private lateinit var cloudinary: Cloudinary;

    override fun onCreate(savedInstanceState: Bundle?) {
        WindowCompat.setDecorFitsSystemWindows(window, false)
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        myImageView = binding.myImageview;

        initCloudinary();

        var url = cloudinary.image().generate("/samples/animals/cat");

        Glide.with(baseContext).load(url).into(myImageView);
    }

    fun initCloudinary() {
        cloudinary = Cloudinary("cloudinary://@dkxtn2tlm")
    }
}